PYTHON_API_CONVENTIONS.md

```diff
@@ -0,0 +1,193 @@
+## Python API Coding Style Conventions
+
+Here is a list with some code conventions used by raylib in python:
+
+| Code element        |             Convention              | Example                                   |
+|---------------------|:-----------------------------------:|-------------------------------------------|
+| Variables           |             lower_case              | `delta_time = 0`, `player_age = 18`       |
+| Local variables     |             lower_case              | `player_position = Vector2(0, 0)`         |
+| Global variables    |             lower_case              | `window_ready = False`                    |
+| Constants variables |              ALL_CAPS               | `MAX_VALUE = 8`, `SCREEN_WIDTH = 800`     |
+| Definitions values  |              ALL_CAPS               | `MAX_BUILDINGS = 5`[^1]                   |
+| string values       |              always ""              | `output = "Hello"`, `"welcome"`           |
+| float values        |             always x.x              | `gravity = 10.0`                          |
+| Operators           |           value1 * value2           | `product = value * 6`                     |
+| Operators           |           value1 / value2           | `division = value / 4`                    |
+| Operators           |           value1 + value2           | `sum = value + 10`                        |
+| Operators           |           value1 - value2           | `res = value - 5`                         |
+| Class               |              TitleCase              | `class TextureFormat`                     |
+| Enum Class members  |              ALL_CAPS               | `PIXELFORMAT_UNCOMPRESSED_R8G8B8`         |
+| Class members       |              lowerCase              | `texture.width`, `color.r`                |
+| Functions           |    lowerCase & wordSeparationBy_    | `init_window()`, `update_camera_center()` |
+| Functions params    |              lowerCase              | `width`, `height`                         |
+| Ternary Operator    | result1 if condition else result2 | `print("yes" if value == 0 else "no")`  |
+[^1] like `macro definitions` of value in C
+
+Some other conventions to follow:
+ - **ALWAYS** initialize all defined variables.
+ - **use TABS / 4 spaces**.
+ - Avoid trailing spaces, please, avoid them
+ - Avoid using **semicolon** as you can
```

Comment:
There are linters and formatters for that. `ruff`, `black` and `isort` will be enough to not bothering with those rules.